lib/IRGen/IRGenSIL.cpp

```diff
@@ -719,6 +719,28 @@ class IRGenSILFunction :
     }
   }
 
+  // To make it unambiguous whether a `var` binding has been initialized,
```

Comment:
///
Doesn't show up in Doxygen, but some editors format the comments nicer then.

Reply:
Sure.